sweepbatcher/sweep_batch.go

```diff
@@ -456,6 +462,11 @@ func (b *batch) addSweep(ctx context.Context, sweep *sweep) (bool, error) {
 	b.log.Infof("adding sweep %x", sweep.swapHash[:6])
 	b.sweeps[sweep.swapHash] = *sweep
 
+	// Update FeeRate.
```

Comment:
nit: a more descriptive comment would be nice, mentioning that the sweeps min fee rate is the basis for fee bumps or something like that.

Reply:
Fixed